css/styles.css

```diff
@@ -47,98 +50,180 @@ nav{
 
 nav ul {
 	float: right;
-	padding: 2% 5%;
+	padding: 0 5%;
 	font-weight: bold;
 }
 
 nav a {
 	text-decoration: none;
-	color: #02C7A6;
+	color: #8F9194;
 }
 
 a:hover {
-	color: #FAC002;
+	color: #5AC4F9;
+}
+
+nav h1 {
+	display: inline;
 }
 
 nav li {
 	display: inline;
-	font-size: 2rem;
-	padding: 20px;
+	padding: 5px 20px;
+	position: relative;
 }
 
-.navColor {
-	width: 100%;
-	height: 10%;
-	background-color: #02C7A6;
+nav li::before {
+	content: " ";
+	height: 30px;
+	width: 90px;
+	background-color: #EFE44D;
 	position: absolute;
-	bottom: 0;
+	top: -15px;
+	z-index: -1;
+}
+
+.aboutNav::before {
+	left: -5px;
+}
+
+.projectsNav::before {
+	left: -5px;
+}
+
+.contactNav::before {
+	left: -5px;
 }
 
 .about {
-	overflow: auto;
+	height: 100vh;
+	position: relative;
+	overflow: hidden;
+}
+
+.aboutImage {
+	width: 100%;
+	position: absolute;
+	bottom: 0;
 }
 
 .about h1 {
-	color: #FFFFFF;
+	font-size: 2rem;
+	text-align: left;
 }
 
 .about div {
 	float: left;
 }
 
 .aboutContentLeft {
-	background-color: #F5313E;
+	background-color: #8F9194;
 	padding: 10% 5% 10% 5%;
-	width: 65%;
+	width: 40%;
 	text-align: justify;
 	color: #FFFFFF;
+	font-size: 1.25rem;
+	height: 90%;
 }
 
 .contentEmphasisLine {
 	text-align: right;
 }
 
+.about .skillsListLineOne::before {
+	content: " ";
```

Comment:
the ::before css has lots of common stuff, I would recommend making some top level generic css and then just changing the width on each item.

Reply:
Done